pkg/apis/core/types.go

```diff
@@ -5654,6 +5657,25 @@ type TopologySpreadConstraint struct {
 	// in their corresponding topology domain.
 	// +optional
 	LabelSelector *metav1.LabelSelector
+	// MinDomains indicates a minimum number of domains.
+	// When the number of domains with matching topology keys is less than `minDomains`,
+	// Pod Topology Spread treats "global minimum" as 0.
+	// As a result, when the number of domains is less than `minDomains`,
+	// scheduler doesn't schedule a matching Pod to Nodes on the domains that have the same or more number of matching Pods as `maxSkew`.
+	// Default value is 1. When value is different from 1, WhenUnsatisfiable must be DoNotSchedule.
+	// This is an alpha field and requires enabling MinDomainsInPodTopologySpread feature.
```

Comment:
It's better to put this at the end of the comment, after an empty `//` line, for better visibility.

Reply:
Sorry, I misunderstood the comment.